src/api/sign/request.js

```diff
@@ -0,0 +1,8 @@
+module.exports = (efx, contents) => {
+  return {
+    // headers: new window.Headers(),
+    // mode: 'cors',
+    // cache: 'default'
+    json: JSON.stringify({contents})
```

Comment:
@plutoegg / @prdn do we really mean `JSON.stringify( { contents } )` and not `JSON.strifiy(contents)` ?